shared/test_run_query.go

```diff
@@ -126,21 +128,23 @@ func (t testRunQueryImpl) LoadTestRunKeys(
 			baseQuery = baseQuery.Filter("Labels =", i.(string))
 		}
 	}
-	var globalKeyFilter mapset.Set
+	var globalIDFilter mapset.Set
 	if len(revisions) > 1 || len(revisions) == 1 && !IsLatest(revisions[0]) {
-		globalKeyFilter = mapset.NewSet()
+		globalIDFilter = mapset.NewSet()
 		for _, sha := range revisions {
 			var ids TestRunIDs
 			if ids, err = loadKeysForRevision(t.store, baseQuery, sha); err != nil {
 				return nil, err
 			}
 			for _, id := range ids {
-				globalKeyFilter.Add(id)
+				globalIDFilter.Add(id)
 			}
 		}
+		log.Debugf("Found %v keys across %v resivions", globalIDFilter.Cardinality(), len(revisions))
```

Comment:
```suggestion
		log.Debugf("Found %d keys across %d revisions", globalIDFilter.Cardinality(), len(revisions))
```

Two nits: 1. `%v`, 2. typo "resivions"

The same applies to the other logs in this file.